cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/CvrSampleListUtil.java

```diff
@@ -137,16 +137,16 @@ public void addNewDmpSample(String sampleId) {
      * @return the portalSamples
      */
     public Set<String> getPortalSamples() {
-        portalSamples.removeAll(samplesInvalidPatientIdList);
+        //portalSamples.removeAll(samplesInvalidPatientIdList);
```

Comment:
On every call to `getPortalSamples(`) we ran `portalSamples.removeAll(samplesInvalidPatientIdList);`  `getPortalSamples()` is called a lot.  Instead, when anything is added to `samplesInvalidPatientIdList`, remove that from the `portalSamples` list.